CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## UNRELEASED
+
+### Adds
+
+* Add support for npm workspaces dependencies. A workspace dependencies can now be used as an Apostrophe module even if it is not a direct dependency of the Apostrophe project. Only direct workspaces dependencies of the Apostrophe project are supported. e.g. I have an Apostrophe project called `website`. `website` is set with 2 workspaces, `workspace-a` & `workspace-b`. `workspace-a` `package.json` contains a module named `blog` as a dependency. `website` can reference `blog` as enabled in the `modules` configuration.
```

Comment:
A workspaces dependencies -> A workspaces dependency

"For instance, I have..."

"With 2 workspaces" -> "with two npm workspaces"

Link "npm workspaces" to relevant npm documentation on the first mention

(And sync up with Bob for docs)